### PR TITLE
Adding Author Information and Submitting Blog Post

### PR DESCRIPTION
**Pull request by brijos**
### Description
[Adding author information to project website]
 
### Issues Resolved
[None]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+With this latest version of the OpenSearch distribution (OpenSearch, OpenSearch Dashboards, as well as plugins and tools) you can enjoy a number of new features and enhancements as well as improvements to stability and efficiency.
+
+One of the major new additions to 1.3.0 is a set of tools for observability between logs, metrics, and other live data. When critical, event-based information is correlated with other application or system data and reported centrally, availability issues are easier to identify and resolve. Try these new tools out (see Observability, below) and let the community know what you think.
+
+Here is a list of major and minor feature enhancements that have been included in this release:
+
+### Index State Management
+
+* **Continuous Mode in Transform**: With this release, you can now run transforms in a continuous mode based on a schedule. This enables you to keep an incremental transform job running on a group of source indices actively ingesting new data. At each run, only the modified buckets will be transformed, reducing the overhead for users who want to their transform up to date.
```
Comment (stockholmux) [inline _posts/2022-03-16-launch-announcement-1-3-0.markdown:24]: 'for users who want to their transform up to date' -> Something is missing here - 'to **keep** their transform up to date' maybe? Also is up to date hypenated?

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+### Observability
+
+Customers can now create custom Observability Applications to view the availability status of all their systems, where log events can be combined with trace and metric data, into a single view of system health empowering developers and IT Ops to resolve issues faster and with fewer escalations.
```
Comment (stockholmux) [inline _posts/2022-03-16-launch-announcement-1-3-0.markdown:28]: 'Customers' is weird in an open source blog. "Users" maybe?

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+![Trace log Analytics]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/trace-log-correlation.gif){: .img-fluid }
+
+* **Live Tail**: Previously, when users were watching a live event take place, they had to manually refresh their view. Users can now configure the interval in which content is refreshed saving the hassle of manually refreshing.
+
+![Live Tail View]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/Live-Tail.gif){: .img-fluid }
+
+* **Field Insights and Enhanced Visualization Support**: Visualization types such as Pie and heat maps were added as well as the ability to add named threshold markers as lines on the visualizations. When users want to know more about a metric that they are looking at, they can select view surrounding events to get a correlated picture. In the event a user needs a reference point for a future meeting, visualizations can now be saved to notebooks for convenience.
```
Comment (stockholmux) [inline _posts/2022-03-16-launch-announcement-1-3-0.markdown:42]: "they can select view surrounding events to get a correlated picture" -> Something doesn't parse for me. Is "view surrounding events" a button or something?

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+### SQL and PPL
+
+* **PPL Runtime Fields**: Users may opt out of formatting the schema upon writing the index (schema-on-write). This opt out can be used to speed up indexing time, or for additional presentation flexibility. Users who want to define their schema upon querying the index (schema-on-read), can now use regex and parse commands to further format their schema using PPL.
+
+![PPL runtime workflow]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/PPL-Parse.gif){: .img-fluid }
+
+
+* **Machine Learning Support in PPL (Piped Processing Language):** Users can now process arbitrary observability events using Anomaly Detection based on Random Cut Forest (RCF) and KMeans commands in PPL.
+* **Improved Aggregate Functions Support in SQL and PPL**: Users expect to use aggregate functions in SQL and PPL. OpenSearch SQL and PPL now supports ORDER BY and default query limit size.
```
Comment (stockholmux) [inline _posts/2022-03-16-launch-announcement-1-3-0.markdown:54]: ORDER BY should be in backticks (``)

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+* **Support for Group Field and Span in Stats Command**: Users can now use group fields together and can span by a specified interval using the Stats command in PPL. Also, users can now create multiple time series’ with a single command.
+* **Comma-Separated Index Matching**: Users can now query multiple indices using a comma separated value using PPL.
+* **CAST Function Supported in PPL**: Users can now change datatypes using the CAST function in PPL.
+* **Support IN Clause in SQL and PPL**: Users can now use the IN clause within SQL and PPL to select from within a value list.
```
Comment (stockholmux) [inline _posts/2022-03-16-launch-announcement-1-3-0.markdown:58]: IN should be in backticks (``)

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+* **Support date_nanos in SQL and PPL**: Users can now search on indices with the field type of data_nanos.
```
Comment (stockholmux) [inline _posts/2022-03-16-launch-announcement-1-3-0.markdown:59]: date_nanos should be in backticks (``)

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+* **Cluster Metric Monitors**: Users can now create monitors within Alerts which help administrators feel more confident about managing the health of their clusters. The new monitor reports on dimensions including, but not limited to CPU usage, JVM memory usage, and total number of documents coming into the cluster.
+
+### Anomaly Detection
+
+* **New Detector Validation**: When setting up a new anomaly detector, the parameters specified will be validated to ensure that the criteria can initialize and that the detector will perform. The validation presented in the user interface and API will prevent detectors from being created which will not initialize.
+
+### ML Commons
+
+* **Machine Learning (ML) Commons:** A new solution that makes it easy to develop new machine learning features. It allows engineers to leverage existing open source machine learning algorithms and reduce the efforts to build them from scratch. It also removes the necessity from engineers to manage the machine learning tasks which will help to speed the feature developing process.
+* **K-Means and Random Cut Forest Algorithm Support**:
```
Comment (stockholmux) [inline _posts/2022-03-16-launch-announcement-1-3-0.markdown:72]: In other part of the document it's written as 'Kmeans'

**Review by stockholmux — CHANGES_REQUESTED** [review]
I found some small issues - see review.

Once done, I'll merge and add:
- Replace images with stills & add click through links for animations
- Insert paragraph about Deb and RPM

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 7 above)
Comment (ylwu-amzn) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:72]: Let's keep consistent, seems "K-means" is mostly used

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 1 above)
Comment (brijos) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:24]: Nice catch.  I believe in this case up-to-date is hyphenated.

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 2 above)
Comment (brijos) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:28]: Moved to Users

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 3 above)
Comment (brijos) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:42]: "View surrounding events" is a link that users can select from.  The UX should be clearer.

image

Visualization types such as pie and heat maps were added as well as the ability to add named threshold markers as lines on the visualizations. When users want to know more about a metric that they are looking at, they can select "View surrounding events" to get a correlated picture.  In the event a user needs a reference point for a future meeting, visualizations can now be saved to notebooks for convenience.

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 4 above)
Comment (brijos) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:54]: Can you show me an example of where it is common to use a backtick for this scenario?

Comment (brijos) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:54]: Got it!

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 5 above)
Comment (brijos) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:58]: Should CAST be included as well?

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 6 above)
Comment (brijos) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:59]: Got it!

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 7 above)
Comment (brijos) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:72]: Got it!

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 5 above)
Comment (stockholmux) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:58]: yeah. CAST as well.

File: _posts/2022-03-16-launch-announcement-1-3-0.markdown (hunk 3 above)
Comment (stockholmux) [reply _posts/2022-03-16-launch-announcement-1-3-0.markdown:42]: that looks good @brijos

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**